db/index.js

```diff
@@ -54,11 +56,71 @@ async function createProduct(product) {
   }
 };
 
+async function createUser({firstName, lastName, email, imageurl, username, password, isAdmin}) {
+  const hashedPassword = await bcrypt.hash(password, SALT_COUNT);
+  try {
+      const { rows: [user] } = await client.query(`
+      INSERT INTO users("firstName", "lastName", email, imageurl, username, password, "isAdmin")
+      VALUES($1, $2, $3, $4, $5, $6, $7)
+      RETURNING id, username;
+  `, [firstName, lastName, email, imageurl, username, hashedPassword, isAdmin]);
+  console.log('createdUser:', user)
+  // delete user.password
```

Comment:
Let's double check whether or not we should return password or not. If not let's remove commented out code.